services/QuillLMS/lib/tasks/gen_ai_tasks.thor

```diff
@@ -422,27 +422,32 @@ class GenAITasks < Thor
 
   desc 'populate_concepts_and_rules', 'Seed the 3 GenAI concepts, the 6 rules needed by the system'
   def populate_concepts_and_rules
-    concept_mapping = {
-      'because' => 'qkjnIjFfXdTuKO7FgPzsIg',
-      'but' => 'KwspxuelfGZQCq7yX6ThPQ',
-      'so' => 'IBdOFpAWi42LgfXvcz0scQ'
+    suborder_mapping = {
+      'because' => { true => 1, false => 2 },
+      'but' => { true => 3, false => 4 },
+      'so' => { true => 5, false => 6 }
     }
-    rules_uids_optimal = Evidence::GenAI::ResponseBuilder::RULES_OPTIMAL
-    rule_uids_suboptimal = Evidence::GenAI::ResponseBuilder::RULES_SUBOPTIMAL
+
+    concept_mapping = Evidence::GenAI::CONCEPT_MAPPING
+    rules_uids_optimal = Evidence::GenAI::RULES_OPTIMAL
+    rules_uids_suboptimal = Evidence::GenAI::RULES_SUBOPTIMAL
 
     Evidence::Prompt::CONJUNCTIONS.each do |conjunction|
-      [true, false].each do |optimal|
-        uid_mapping = optimal ? rules_uids_optimal : rule_uids_suboptimal
+      [true, false].each do |is_optimal|
+        uid_mapping = is_optimal ? rules_uids_optimal : rules_uids_suboptimal
 
-        Evidence::Rule.create(
+        rule = Evidence::Rule.create(
           uid: uid_mapping[conjunction],
           concept_uid: concept_mapping[conjunction],
-          universal: true,
+          universal: false,
           rule_type: Evidence::Rule::TYPE_GEN_AI,
-          optimal:,
+          optimal: is_optimal,
           state: 'active',
-          name: "GenAI universal #{conjunction} - #{optimal ? 'optimal' : 'suboptimal'}"
+          suborder: suborder_mapping[conjunction][is_optimal],
+          name: "GenAI #{conjunction} - #{is_optimal ? 'optimal' : 'suboptimal'}"
         )
+
+        Prompt.where(conjunction:).each { |prompt| rule.prompt_rules.create(prompt:) }
```

Comment:
Do we want to scope this to just prompts associated with a `GenAI` activity?

Reply:
Yes, that's a good idea.